sessionctx/variable/session.go

```diff
@@ -1354,8 +1341,20 @@ func (s *SessionVars) SetAlloc(alloc chunk.Allocator) {
 }
 
 // ClearAlloc indicates stop reuse chunk
-func (s *SessionVars) ClearAlloc() {
+func (s *SessionVars) ClearAlloc(cc *chunk.Allocator, err bool) {
```

Comment:
err is usually an abbreviation for error

Reply:
done